appveyor.yml

```diff
@@ -39,6 +39,9 @@ install:
   - 7z x scons-2.5.1.zip
   - cd scons-2.5.1
   - python setup.py install
+  - echo *** Patching NSIS
+  - cd %APPVEYOR_BUILD_FOLDER%
+  - patch "C:\Program Files (x86)\NSIS\Include\MultiUser.nsh" < build\appveyor\NSIS.patch
```

Comment:
1. Yes, if that fails, the whole build fail. I tried it ;)
2. I'm not aware of any 64 bits NSIS version available. I'm pretty sure there is only one flavour of NSIS 3. This could fail in NSIS 4 (if we have that one day), but not without changing the  build environment image

and 3. this ugly hack will disappear with wix integration PR#862 (1yr today \o/)